.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @BenMMcLean @pdsla112
```

Comment:
Why is this, and all the dependency/license changes included in your PR?

Reply:
I believe my branch was not up to date, and I pulled it from the main branch, which is why it might be included.

Reply:
Fair enough, I'm going to assume this will be resolved whenever this is merged